docs/docs/03-syntax-and-usage/11-script-templates.md

```diff
@@ -103,3 +103,50 @@ func main() {
 	}
 }
 ```
+
+You can also directly render the Javascript function, passing in Go data, using the `!` expression:
```

Comment:
Is it worth calling out that `ScriptComponents` are templ Components, and can therefore be included in the page?

Aside from that, it probably is worth explaining that the JavaScript function code will be written once to the page, but that the function will be called every time its parent component is rendered.

An example showing this (with output) would be very useful.

Reply:
With the new output, I updated the example to call the script twice, to show it's emitted once.